Changes

```diff
@@ -246,6 +246,8 @@ Working version
   raise `Sys_error` in this case.
   (Nicolás Ojeda Bär, review by Gabriel Scherer)
 
+- #12267: Fix stack alignment computation
```

Comment:
You should include the present PR number and credit yourself, the reporter, and eventually any reviewer.

Reply:
I'm waiting for the review to happen to amend this.